dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -420,123 +389,162 @@ class CompileTest {
 
 /// Measure application memory usage.
 class MemoryTest {
-  const MemoryTest(this.testDirectory, this.packageName, { this.testTarget });
+  MemoryTest(this.project, this.test, this.package);
+
+  final String project;
+  final String test;
+  final String package;
+
+  Future<void> get ready => _ready?.future;
```

Comment:
I find this name a bit confusing, e.g a `ready` field of a MemoryTest object sounds to me like something that completes once, when the test is read (to start).

This should probably get some documentation, I'd also consider a different name, I don't have a very good suggestion though, maybe something in the spirit of nextMessageFuture?

Reply:
Renamed "receiveNextMessage", added docs.